parm/atm/geom/background.yaml

```diff
@@ -0,0 +1,18 @@
+fms initialization:
+  namelist filename: '$(fv3jedi_fix_dir)/fmsmpp.nml'
+  field table filename: '$(fv3jedi_fix_dir)/field_table'
+akbk: '$(fv3jedi_fix_dir)/akbk.nc4'
+# input.nml
+layout:
+  - $(layout_x)
+  - $(layout_y)
+io_layout:
+  - $(io_layout_x)
+  - $(io_layout_y)
+npx: $(npx_ges)
```

Comment:
This file is identical to parm/atm/geom/analysis.yaml except for npx and npy.   The analysis yaml references _anl variables while the background yaml references _ges variables.   Can we combine these two yamls into one with a python script inserting them in the correct location with the correct values in the monolithic yaml passed to fv3jedi_var.x?    The motivation for this is to reduce redundancy.